tests/integration/server.py

```diff
@@ -231,6 +233,18 @@ def _custom():
     return json.dumps(tests), 200
 
 
+@app.route('/environment', methods=['GET'])
+def _check_environment():
+    # determine what cloud env we're running in; essentially, GAE vs GKE
+    # for GAE, we'll check the existence of an env var starting with 'GAE_'
+    # if none exist, assume we're in GKE
+    env = os.environ
```

Comment:
Can you match the logic here: https://github.com/GoogleCloudPlatform/google-cloud-python/blob/11e310ab9f63f06252ff2b9ada201fd8c11ce06c/logging/google/cloud/logging/client.py#L307

Pulling this logic into a standalone lib so that everything can just reference it is a TODO.

Reply:
done